build/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM node:8
+RUN apt-get update && apt-get install -y make gcc nasm libpng-dev
```

Comment:
Need libpng-dev as part of the webpack asset pipeline. To avoid having to build this Docker image each time, I've manually pushed the built version to https://hub.docker.com/r/mattermost/mattermost-build-webapp/. We could automate this, I suppose, but for now I'm anticipating we pin to the 5.2.0 tag until the base image has to change.